docs/synthetics/private-locations.md

```diff
@@ -0,0 +1,37 @@
+title: Private Locations
+description: Private locations allows you to monitor the APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc
+
+Private locations allow you to run the synthetics tests from your private network. Some of the use cases for using private locations are:
+
+* Monitor internal APIs or web applications that are not accessible from the Internet
+* Compare the performance of your applications between the Internet & your private network
+* Monitor your applications from specific locations that are not supported by Sematext
+
+## Create Private Location
+
+To create a private location, navigate to Synthetics -> Private Locations -> New Private Location. 
+
+![Create Private Location]()
+
+* **Name** - Unique name for the private location
+* **Description** - A brief description of the location
+* **City** - Select the city where the machine running the agent is physically located. This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city.
+
+Each private location is assigned a unique token. You can disable or delete a location, when not in use.
+
+## Install Private Agents
+
+After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images one for each HTTP & Browser monitor respectively. 
```

Comment:
There are two Docker images one for each HTTP & Browser monitor respectively.
=>
There are two Docker images, one for HTTP monitor and another for Browser monitor.